RandomizerCore/Hyrule.cs

```diff
@@ -212,8 +213,11 @@ public Hyrule(RandomizerConfiguration config, BackgroundWorker worker, bool save
 
         bool raftIsRequired = IsRaftAlwaysRequired(props);
         bool passedValidation = false;
+        HashSet<int> freeBanks = new();
         while (palaces == null || palaces.Count != 7 || passedValidation == false)
         {
+            freeBanks = new(ROM.FreeRomBanks);

```

Comment:
Currently not used but defined here so future changes can use some of the banks for additional rooms.